docs/source/diagrams/ReflectometryReductionOne_LowLvl-v1_wkflw.dot

```diff
@@ -55,17 +66,28 @@ $algorithm_style
   rebin           [label="Rebin"]
   scale           [label="Scale"]
   calcResolution  [label="CalculateResolution"]
+  convertDet      [label="Convert Units\n(AlignBins = True)"]
+  convertMon      [label="Convert Units"]
+  groupDet        [label="Group Detectors"]
+  cropDetWS       [label="Crop Detector\nWorkspace"]
+  rebinDet        [label="Rebin Detectors"]
+  cropMonWS       [label="Crop Monitor\nWorkspace"]
+  calcFlatBg      [label="Calculate Flat\nBackground"]
+  rebinToWS       [label="Rebin To\nWorkspace"]
```

Comment:
Again, those should be `CropWorkspace`, `CalculateFlatBackground`, and `RebinToWorkspace`, which are the exact names of the algorithms we are running.